### PR TITLE
theming now can be used for the date time picker

### PR DESCRIPTION
**Pull request by commitimpush**
Enabled theming for date time picker

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -25,12 +25,12 @@ class RosterDemo extends StatelessWidget {
...
                 Text(
                   'Team Rooster',
-                  style: Theme.of(context).textTheme.headline6,
+                  style: Theme.of(context).textTheme.bodyLarge,
                 ),
                 const Spacer(),
                 Text(
                   'Persoonlijk Rooster',
-                  style: Theme.of(context).textTheme.headline6,
+                  style: Theme.of(context).textTheme.bodyLarge,
```
Comment (Gorter-dev) [inline example/lib/main.dart]: No context related theming as default.

File: example/lib/main.dart [hunk 2]
```diff
@@ -25,12 +25,12 @@ class RosterDemo extends StatelessWidget {
               children: [
                 Text(
                   'Team Rooster',
-                  style: Theme.of(context).textTheme.headline6,
+                  style: Theme.of(context).textTheme.bodyLarge,
                 ),
                 const Spacer(),
                 Text(
                   'Persoonlijk Rooster',
```
Comment (Gorter-dev) [inline example/lib/main.dart]: No hardcoded strings

**Review by Gorter-dev — CHANGES_REQUESTED** [review]

File: example/lib/main.dart (hunk 1 above)
Comment (commitimpush) [reply example/lib/main.dart]: fixed

File: example/lib/main.dart (hunk 2 above)
Comment (commitimpush) [reply example/lib/main.dart]: fixed

**Review by Gorter-dev — APPROVED** [review]

**Pull request merged**